### DISCUSSION TITLE
v1.1.0-pre

### DISCUSSION THREAD

**Discussion by kounoike** [kounoike/obs-virtualbg#45] [Announcements]
## 🚀 Features

- add DirectML DLL into zip @kounoike (#43)
- Feature: add gpu cpu settings @kounoike (#39)
- add Linux suppport @kounoike (#33)
- add mac os support @kounoike (#30)

## 🐛 Bug Fixes

- fix auto scheduler support @kounoike (#44)
- Fix Mac OS X build error @kounoike (#38)
- Fix problem partially, when filter\_video calls with frame->width or height == 0 @kounoike (#35)

## 🔧 Refactoring

- Refactor split functions, more error detection @kounoike (#37)

<hr /><em>This discussion was created from the release <a href='https://github.com/kounoike/obs-virtualbg/releases/tag/v1.1.0-pre'>v1.1.0-pre</a>.</em>

**Comment by true-Grand** [comment]
Is there instructions for macOS? I can't seem to do it.
